client/my-sites/theme/main.jsx

```diff
@@ -326,14 +321,26 @@ const ThemeSheet = React.createClass( {
 		return <ThemeDownloadCard theme={ this.props.id } href={ this.props.download } />;
 	},
 
+	getDefaultOptionLabel() {
+		const { defaultOption, active: isActive, isLoggedIn, price } = this.props;
```

Comment:
> Is there any reason for assigning value from `active `to`isActive` except the fact that `is` better meaning.

Yeah, there's also `isLoggedIn`, so I was striving for consistency. Also, keep in mind that we're going to rely on a `isActiveTheme` selector for this in the near future :-)

> Maybe just update `active` name in `props`?

This comes from the `theme` object as obtained from the API, so we'd need to change it in the reducer. Not worth it since we're not going to use it for very much longer anyway :-)